docs/participate/online-communities.mdx

```diff
@@ -8,46 +8,40 @@ The Mina community has many online groups where you can connect with other Mina
-
-[#grants](https://discord.gg/AcFCvVvfVW) — to discuss grant opportunities
+- [#announcements](https://discord.com/channels/484437221055922177/601171209287368715) to follow announcements
+- [#zkapps-general](https://discord.com/channels/484437221055922177/910549624413102100) to ask general questions about zkApps, how to use a zkApp, and so on
+- [#zkapps-developers](https://discord.com/channels/484437221055922177/915745847692636181) to meet other developers building zkApps with SnarkyJS
+- [#mentor-nodes](https://discord.com/channels/484437221055922177/746316198806814760) to get help from the community on setting up and running nodes
+- [#mainnet-block-producers](https://discord.com/channels/484437221055922177/799597981762453535) to meet other node operators and block producers
+- [#grants](https://discord.com/channels/484437221055922177/727960609832042607) to discuss grant opportunities
 
 ## Discussion Forums
 
-[Mina Research Forums](https://forums.minaprotocol.com/)
-
-[Mina on Github Discussions](https://github.com/MinaProtocol/mina/discussions) — to discuss new Mina releases
+- [MinaResearch](https://forums.minaprotocol.com/)
+- [MinaProtocol/mina GitHub](https://github.com/MinaProtocol/mina/discussions)
 
 ## Social Media
 
-[Twitter - @minaprotocol](https://twitter.com/minaprotocol)
-
-[Telegram - @minaprotocol](https://t.me/minaprotocol)
-
-[Telegram - @MinaProtocolOfficialNews (Announcements Only)](https://t.me/MinaProtocolOfficialNews)
-
-[Reddit - /r/minaprotocol](https://www.reddit.com/r/MinaProtocol/)
+- Twitter [@minaprotocol](https://twitter.com/minaprotocol)
+- Telegram [@minaprotocol](https://t.me/minaprotocol)
+- Telegram [@MinaProtocolOfficialNews](https://t.me/MinaProtocolOfficialNews)
+- Reddit [/r/minaprotocol](https://www.reddit.com/r/MinaProtocol/)
+- YouTube [@minaprotocol](https://www.youtube.com/c/minaprotocol)
 
-## YouTube
+## Mina Blogs
 
-[YouTube - Mina Foundation](https://www.youtube.com/c/minaprotocol)
+- [Mina Protocol Blog](https://minaprotocol.com/blog)
+- [Mina Protocol Medium](https://medium.com/@minaprotocol)
 
-[YouTube - Evan Shapiro, CEO of Mina Foundation](https://www.youtube.com/channel/UCZebnzqd__m_lHP_d5qgsoQ)
+## Communities in the Mina Ecosystem
 
-[YouTube - O(1) Labs](https://www.youtube.com/channel/UCrT2KUIIrDPJk8leVu7GYdQ)
+- [Mina Community Blog](https://minafoundation.notion.site/minafoundation/Mina-Community-Blog-91b8fdfd1b9e444ca55ddd1b90bade3c)
 
-## Blogs
+    You can be an active contributor to collective brilliance by sharing links to community-generated blogs, videos, articles, and other resources. For inspiration and steps on how to contribute to the experimental community blog, see [Open Call for Mina Community Creators](https://minaprotocol.com/blog/mina-community-blog).
 
-[Medium - @minaprotocol](https://medium.com/@minaprotocol)
+- [O(1) Labs](https://blog.o1labs.org/) incubators of the Mina Protocol
```

Comment:
```suggestion
- [O(1) Labs Blog](https://blog.o1labs.org/) - incubators of the Mina Protocol.
```
Suggestion to highlight "Blog" so it's clear why these links exist here b/c they're content related.